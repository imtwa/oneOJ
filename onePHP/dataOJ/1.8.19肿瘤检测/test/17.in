76
70 65 11 27 32 44 89 96 26 61 92 27 71 89 14 49 37 98 93 25 74 57 18 20 89 18 10 66 69 34 28 69 47 94 24 46 12 90 84 20 99 11 24 86 61 21 46 29 94 87 57 50 24 19 13 45 34 78 18 73 95 13 89 69 87 46 77 51 93 62 11 48 51 36 44 98
98 63 58 97 33 19 84 100 93 66 48 65 92 33 56 21 62 60 99 33 24 50 27 65 45 34 88 24 66 64 57 82 24 43 16 18 93 30 62 22 86 11 27 96 91 49 83 91 30 83 56 53 62 29 48 58 90 87 28 89 57 21 73 30 94 78 20 19 91 86 17 79 33 98 31 93
81 70 37 85 100 78 85 74 28 21 50 57 12 28 64 71 25 54 72 47 98 56 44 58 70 17 29 14 91 28 31 76 55 26 30 97 92 13 74 74 74 25 99 36 37 45 82 28 48 96 81 37 74 10 31 60 48 65 40 32 62 82 80 15 13 62 93 52 20 47 93 64 59 93 89 82
30 23 46 56 12 75 13 87 66 53 84 51 19 26 69 72 69 90 77 99 63 69 45 49 42 19 79 67 24 37 77 62 64 17 26 54 19 85 70 72 42 26 77 10 61 71 13 92 86 63 59 60 41 28 71 58 99 43 72 38 80 15 20 55 95 30 22 39 12 43 62 52 31 36 92 30
65 24 17 33 57 55 38 81 52 43 83 68 61 86 16 84 54 26 98 43 25 26 39 92 57 73 46 73 92 51 37 35 84 93 70 10 46 30 17 58 91 75 59 23 50 18 36 33 60 59 80 14 55 31 11 87 67 41 20 87 19 100 60 92 10 15 44 26 88 25 15 86 79 63 83 60
91 20 21 77 94 12 33 85 47 78 19 96 71 51 72 37 71 67 51 65 42 47 79 27 50 16 96 36 32 75 21 35 45 12 46 95 84 24 49 40 65 19 25 39 82 92 27 11 50 28 57 18 78 74 86 86 23 19 55 16 57 85 44 81 36 20 65 50 26 23 71 84 62 55 70 28
38 99 27 49 52 99 66 37 83 93 14 77 20 52 95 36 79 49 68 37 15 77 44 15 39 80 18 88 70 52 41 95 64 82 65 56 85 73 14 12 90 35 17 11 51 78 82 19 97 81 51 19 66 21 41 30 63 35 34 91 27 39 23 29 26 64 94 31 63 38 15 61 78 22 50 80
92 78 18 54 46 94 78 31 58 71 25 74 54 61 15 27 89 77 34 92 88 17 76 71 11 30 91 24 50 42 68 35 44 99 73 28 57 20 17 95 48 72 20 73 59 34 27 28 49 17 44 63 11 29 26 29 33 63 24 78 51 97 71 14 65 96 94 13 68 24 83 58 19 39 66 12
59 41 63 16 92 26 71 26 54 100 60 76 62 58 67 46 93 62 31 36 23 30 78 35 70 53 81 37 27 46 65 65 31 85 17 94 78 17 30 85 29 91 24 62 93 100 49 18 32 79 88 71 70 16 24 71 54 83 50 61 43 76 57 15 27 37 34 37 12 16 26 40 83 58 88 31
50 33 28 22 19 77 93 73 33 89 69 51 10 100 72 50 80 93 56 49 67 95 19 44 89 78 76 31 22 62 53 79 27 94 31 93 91 33 17 72 55 51 85 55 86 58 30 73 30 46 25 43 24 41 99 20 86 47 75 84 20 17 59 90 89 11 74 75 74 29 65 68 42 74 49 13
90 90 77 56 85 64 59 13 60 41 62 92 67 20 18 17 46 81 23 37 82 82 10 45 71 69 61 97 80 34 43 64 93 51 36 11 96 10 33 64 96 30 57 48 72 79 34 50 85 48 33 95 33 88 73 15 50 36 29 19 46 20 48 27 48 59 100 99 97 26 12 47 31 84 38 79
39 80 43 60 52 56 42 59 86 39 79 98 15 60 91 21 49 36 61 77 32 74 99 11 21 69 86 55 44 55 67 90 83 67 81 82 32 38 52 35 91 96 98 29 82 17 58 34 39 60 37 98 69 94 21 12 41 64 50 75 98 27 20 23 43 45 63 54 54 43 65 72 91 92 99 10
98 28 67 64 46 86 55 61 49 15 19 29 66 51 62 26 88 62 70 46 42 10 32 19 40 67 30 97 82 35 74 93 89 25 41 48 11 96 33 57 32 80 89 50 25 72 60 43 25 63 83 63 81 41 49 96 76 82 53 76 63 58 67 64 32 28 43 93 18 60 83 62 85 74 48 98
49 36 76 22 90 43 16 29 29 35 71 48 40 92 88 48 61 18 68 61 71 97 63 33 30 59 89 40 62 32 54 90 84 83 11 29 36 96 89 28 94 16 75 27 24 69 96 39 49 76 90 20 62 83 36 41 34 45 58 23 22 34 82 41 34 59 82 77 22 83 19 97 59 36 59 15
47 42 66 61 30 22 13 65 16 64 10 16 10 39 44 78 90 27 22 20 93 26 14 64 51 25 98 82 47 37 22 28 73 87 85 32 35 73 36 62 79 99 26 52 46 80 55 13 82 81 74 85 97 85 75 33 29 13 33 44 96 54 58 67 48 68 36 74 12 25 84 50 40 54 70 82
84 86 82 24 40 82 78 34 96 26 85 59 37 63 41 38 70 82 23 96 29 74 12 30 73 34 58 34 100 91 56 77 26 39 27 95 55 30 30 38 56 23 24 32 38 85 98 88 54 83 86 66 62 38 33 60 49 75 85 67 26 78 87 91 37 95 63 25 11 15 18 87 39 31 23 37
84 93 98 21 60 13 22 59 97 43 88 53 23 10 62 38 84 25 87 97 91 26 99 62 85 85 83 31 56 19 72 49 84 87 88 11 24 46 10 34 30 51 70 69 75 35 31 65 57 67 21 46 19 52 15 64 85 25 24 13 88 54 66 86 27 13 40 69 61 93 93 51 19 83 37 72
98 33 70 90 80 85 39 20 17 14 91 60 94 10 73 23 23 39 11 34 68 90 54 91 45 28 19 18 90 15 36 28 18 38 11 81 69 16 87 90 65 37 18 23 70 72 93 65 33 76 47 68 46 64 96 36 87 58 19 86 56 76 95 83 46 38 20 14 82 92 10 14 50 11 88 27
30 60 20 28 14 32 31 76 87 87 30 98 73 73 58 19 64 21 76 45 55 70 62 18 33 35 96 91 37 61 69 17 63 30 68 33 100 53 97 77 80 16 37 43 19 58 94 44 60 98 20 68 60 95 72 83 86 42 51 11 75 17 68 82 35 92 80 12 65 21 96 74 65 79 75 41
41 39 50 95 80 70 84 41 38 23 38 71 96 99 73 95 23 23 94 16 45 18 87 70 69 36 26 33 57 90 87 12 85 17 25 39 83 39 14 82 43 39 76 59 69 35 76 52 96 58 40 79 98 61 93 12 82 65 35 27 76 78 65 56 54 96 83 39 21 11 12 23 29 81 17 16
14 46 62 61 49 63 60 18 73 15 90 20 73 24 100 51 47 38 10 27 98 97 21 62 48 79 22 34 62 50 50 38 77 74 72 76 24 59 42 29 64 76 72 27 81 46 26 68 82 21 70 17 85 54 10 32 83 75 34 46 31 71 23 21 50 68 58 10 25 88 30 50 91 44 86 93
41 51 63 92 12 67 57 92 78 79 99 59 44 26 41 50 99 20 84 54 99 75 67 17 52 18 67 100 48 28 33 67 81 73 68 32 55 25 77 81 79 19 71 98 76 16 24 41 15 42 20 73 87 59 91 82 39 68 94 67 41 87 91 99 56 93 91 12 23 49 91 54 30 59 93 44
19 62 90 52 29 33 41 70 58 70 91 16 71 58 33 78 71 99 58 87 88 33 91 14 29 65 48 27 18 51 47 28 36 52 86 21 64 74 65 15 97 67 24 72 47 66 63 15 20 64 28 18 29 11 10 82 23 45 36 80 31 77 12 46 35 47 93 20 68 47 26 86 33 88 13 85
33 26 58 67 48 76 29 57 75 29 82 22 72 66 51 57 64 33 76 35 94 11 12 40 52 30 30 90 55 75 27 10 73 33 98 70 98 69 50 55 93 92 53 85 66 87 57 56 49 88 44 55 90 46 78 59 100 56 28 51 56 20 89 62 51 99 72 26 36 31 16 61 22 90 39 31
38 64 29 95 41 72 70 41 84 77 98 65 14 45 96 96 68 19 40 41 96 64 14 92 46 30 24 27 99 97 63 73 30 94 51 23 54 71 77 39 74 100 38 59 99 26 46 90 95 24 26 100 82 58 14 75 24 41 84 44 44 34 29 79 22 20 26 82 16 99 12 14 43 33 30 48
92 81 87 16 73 36 12 27 53 85 24 37 10 11 32 80 26 80 14 53 29 15 27 76 86 25 69 90 41 43 53 35 52 35 73 99 97 58 86 37 91 18 83 98 66 63 78 54 34 25 36 50 27 77 11 34 98 89 15 65 18 13 71 10 60 74 73 51 94 55 15 86 75 11 59 10
86 80 92 66 36 69 82 22 38 74 41 68 82 22 19 89 45 24 14 38 55 11 60 37 12 27 70 91 28 76 36 94 51 63 100 81 26 30 74 90 54 17 64 47 12 63 56 11 79 11 100 16 11 45 32 78 17 99 19 100 31 37 46 74 23 91 62 47 89 83 94 20 34 49 85 93
75 42 18 10 22 47 81 19 38 79 27 68 73 41 84 15 81 93 59 84 43 29 68 14 17 53 96 62 51 10 54 34 61 42 90 34 56 10 82 88 64 58 89 100 34 19 76 34 42 82 22 26 70 13 77 31 63 59 92 43 41 56 38 88 48 30 25 41 69 92 97 63 50 21 37 11
33 18 50 30 12 59 22 44 40 89 35 61 83 46 29 26 89 39 10 100 60 89 25 26 18 85 60 26 74 41 76 48 63 33 13 30 72 90 32 52 39 72 24 70 16 83 35 81 51 77 12 62 93 97 20 52 31 48 92 45 66 70 94 48 72 60 70 64 38 24 54 63 40 44 55 77
63 22 28 28 99 65 17 98 77 11 74 85 37 49 18 53 35 59 31 63 11 43 88 69 79 36 19 22 89 31 10 48 59 55 18 44 59 56 12 45 15 78 25 73 23 32 100 18 16 28 46 20 41 36 15 75 24 52 90 74 92 37 13 84 57 61 79 72 90 92 51 10 38 36 95 76
24 79 71 46 78 45 62 85 89 26 22 52 27 65 93 30 30 41 11 28 89 74 59 50 89 52 91 78 39 29 10 95 32 59 21 91 62 59 70 72 63 40 69 53 56 25 60 74 34 19 26 99 61 86 62 59 57 91 75 85 76 55 91 15 56 61 66 75 31 79 95 40 29 45 53 22
79 77 56 58 22 73 38 97 59 24 23 59 21 17 11 24 63 23 29 89 17 69 61 63 97 36 41 73 50 85 57 16 42 56 83 76 93 60 17 72 79 11 93 63 45 54 26 52 20 21 36 66 68 45 87 33 64 17 80 25 94 82 54 50 63 79 68 33 87 41 60 39 45 42 50 62
51 54 48 35 88 96 82 99 79 40 48 86 95 36 92 65 76 62 100 36 32 31 93 64 82 52 26 27 69 35 93 59 74 54 91 77 36 38 19 27 82 57 82 73 69 67 37 59 22 36 96 82 100 85 87 48 31 15 100 91 43 92 97 98 85 86 78 81 66 20 15 56 27 59 46 22
89 12 70 48 83 63 60 35 10 59 50 93 75 38 86 100 97 25 26 20 44 98 67 33 62 73 89 29 84 79 15 84 32 68 30 21 19 53 89 54 26 93 32 74 25 15 73 75 67 17 88 46 70 34 86 17 11 82 81 35 63 59 58 19 33 22 17 96 26 15 92 50 59 86 15 13
21 23 74 87 50 25 38 56 67 25 11 98 99 98 65 21 75 24 45 52 72 78 11 97 13 86 29 97 93 46 26 31 91 63 35 12 93 59 98 65 96 11 48 29 35 64 76 87 61 80 52 17 33 57 77 55 21 62 44 54 14 14 74 39 75 44 44 28 65 74 81 80 97 78 67 83
25 67 32 41 19 60 77 67 60 98 42 74 59 27 62 33 34 34 87 33 26 28 26 69 96 16 58 66 72 34 87 78 69 64 25 55 74 68 61 79 34 46 24 56 72 50 65 43 75 75 47 95 51 96 48 93 99 48 100 59 33 14 84 33 69 65 63 50 74 87 95 88 27 88 55 10
90 96 37 34 76 83 18 63 70 39 43 80 70 53 38 94 32 91 35 47 12 76 25 85 44 54 87 16 94 50 95 69 47 39 71 20 48 58 90 28 45 47 95 54 78 63 30 32 82 65 49 50 93 74 97 71 71 71 16 87 95 67 47 28 15 71 31 33 32 82 79 70 41 98 68 41
78 99 97 65 16 86 36 20 11 55 93 71 100 37 67 51 53 89 35 67 33 65 72 75 42 19 92 80 24 96 25 62 33 20 31 88 42 75 67 50 58 37 90 75 52 78 29 66 86 100 64 49 81 49 59 10 51 78 54 34 83 100 41 92 14 51 90 96 97 56 13 25 21 63 38 97
35 31 53 15 23 94 68 71 20 21 61 20 56 36 82 37 35 42 75 43 62 90 39 75 27 56 79 88 38 85 46 53 25 54 37 25 44 49 33 79 13 14 61 46 50 24 22 88 94 60 58 80 10 77 67 66 10 39 34 97 21 15 59 83 24 61 41 100 66 75 71 32 60 48 57 43
34 89 68 95 80 35 50 25 70 77 61 14 71 23 96 73 84 27 71 52 17 98 93 67 58 89 35 72 39 26 37 16 14 25 26 25 53 73 67 53 93 52 55 88 100 45 73 61 54 14 95 16 80 90 90 22 25 100 91 20 53 64 88 76 58 17 80 11 20 14 87 15 61 30 88 86
46 42 70 45 17 86 12 95 50 41 29 74 74 25 98 46 80 29 12 59 86 98 17 61 100 61 18 90 55 95 45 54 53 41 32 53 50 22 27 99 56 87 82 89 89 42 17 45 42 44 22 48 99 70 52 64 56 13 16 66 46 95 14 26 72 14 85 98 41 66 16 59 22 26 28 19
38 97 11 62 55 23 55 72 95 35 51 26 83 24 56 10 44 80 73 23 76 15 13 36 20 40 11 41 83 91 63 78 68 67 38 76 71 61 53 53 11 30 35 50 43 39 17 48 79 81 27 96 81 57 39 17 95 40 90 48 97 88 38 50 13 12 78 64 22 32 76 64 70 61 91 37
29 40 28 13 99 89 54 83 21 35 97 55 11 38 77 60 12 67 47 30 49 80 93 39 37 12 38 14 13 70 48 96 51 30 24 18 73 36 35 31 42 56 26 49 94 92 93 65 80 86 28 11 68 54 15 64 91 18 97 77 92 84 36 100 68 20 22 71 68 76 95 59 62 35 68 45
10 62 45 68 71 25 91 55 45 65 25 81 34 29 98 16 32 12 42 38 77 11 44 47 88 27 82 21 51 11 24 35 59 39 44 11 95 88 32 19 34 24 86 31 32 73 27 79 28 40 18 92 34 27 84 55 74 83 92 55 25 17 19 44 78 98 91 28 40 14 64 71 47 18 78 96
50 61 79 64 16 21 23 57 73 17 37 71 91 73 71 23 61 50 86 19 60 46 93 57 11 23 51 93 96 72 23 52 69 66 13 56 39 37 16 82 37 99 72 73 13 73 51 55 76 61 37 89 66 95 84 39 43 100 37 72 78 100 46 86 53 64 33 14 59 55 68 39 88 79 40 31
75 96 29 65 97 47 82 42 38 56 28 67 50 44 21 72 73 97 54 64 75 87 90 82 63 92 17 83 20 87 95 94 51 89 23 26 37 49 77 65 74 73 86 69 21 97 48 42 93 92 49 18 28 36 63 40 93 17 39 56 70 15 47 53 92 17 42 52 53 16 71 62 82 85 10 44
66 82 87 53 40 72 79 52 29 16 28 68 55 47 63 78 58 45 72 44 67 36 88 16 38 83 61 21 72 40 84 38 35 72 54 80 52 86 86 61 71 20 12 56 48 87 75 20 30 76 81 51 19 79 90 99 88 47 22 17 37 58 31 83 24 100 60 57 97 51 16 50 63 46 89 14
33 81 60 26 53 100 60 61 16 93 82 97 38 80 68 100 19 50 45 47 22 67 99 54 35 84 54 25 98 14 26 61 67 89 97 24 67 53 81 13 16 62 90 31 77 11 82 89 46 88 66 90 59 86 46 58 57 46 63 33 78 65 20 74 82 92 41 43 47 71 97 82 98 74 61 46
66 49 42 93 13 56 39 21 24 73 36 37 69 49 12 13 36 60 71 89 65 89 65 54 96 15 11 58 93 29 39 37 67 63 88 49 35 40 10 87 58 14 17 71 74 100 84 86 16 25 41 89 55 21 75 17 90 81 17 59 65 79 21 58 75 97 37 31 63 90 99 62 33 31 91 76
92 68 96 80 89 65 100 66 69 85 77 77 16 84 89 86 92 48 56 11 86 15 85 96 16 73 74 90 27 66 84 32 69 59 57 34 26 100 79 83 50 17 53 34 51 58 17 26 56 19 32 46 90 80 97 13 95 66 100 45 73 74 74 13 56 41 70 21 77 51 96 34 59 62 95 90
98 49 65 66 48 51 18 83 13 62 98 50 81 56 73 84 72 41 58 56 20 75 58 18 95 20 88 86 53 98 13 76 13 77 19 95 23 98 86 95 12 58 59 55 65 49 20 53 80 22 77 76 46 51 50 15 83 79 85 51 90 50 19 58 47 87 64 33 92 60 33 27 97 75 94 76
57 62 53 75 34 85 69 39 73 80 25 49 88 74 91 94 94 40 22 92 49 89 43 79 74 38 50 33 10 51 61 31 52 80 75 73 16 20 37 87 49 61 94 16 93 74 95 15 95 60 88 61 21 19 98 95 41 36 98 41 66 28 52 11 81 38 28 47 49 45 12 56 32 37 12 45
58 17 43 47 91 35 85 94 68 22 15 62 42 62 30 86 56 16 12 85 63 80 58 60 97 53 83 42 83 95 82 70 74 86 50 86 18 43 96 73 56 98 71 19 82 87 91 27 72 19 68 67 53 95 40 96 53 16 42 12 14 25 34 51 31 65 32 15 47 76 17 91 22 68 19 82
46 22 98 66 66 52 35 68 21 58 24 42 90 17 19 55 59 88 52 87 61 76 25 77 72 33 67 88 51 76 72 87 17 92 63 77 65 13 73 42 78 99 83 57 23 51 30 26 84 97 35 67 45 93 31 38 72 13 90 49 80 67 100 75 67 54 96 50 95 81 76 86 26 80 56 81
92 77 76 27 27 58 84 22 58 98 34 47 21 17 57 29 21 77 100 28 48 24 88 62 82 77 69 74 50 20 37 28 42 57 85 34 35 11 28 33 91 64 26 64 36 24 58 39 21 20 12 71 70 15 61 54 99 81 51 18 99 60 30 50 16 29 92 59 93 14 86 75 38 55 95 62
70 53 49 63 94 33 90 70 23 90 14 35 65 15 26 92 49 30 10 75 79 24 50 18 12 86 93 40 60 62 52 19 78 100 82 82 36 28 75 51 30 28 82 41 41 67 66 22 52 20 22 59 27 66 91 53 79 71 84 95 65 12 98 87 11 31 79 71 13 47 40 80 14 95 53 26
74 86 15 65 51 16 18 93 94 73 93 64 95 53 91 10 57 78 54 14 28 33 52 57 90 35 65 14 17 12 88 12 37 18 82 64 82 69 55 92 29 11 27 20 57 19 56 81 97 31 37 79 16 76 87 10 16 27 40 83 20 34 41 19 93 71 18 49 15 19 47 40 83 96 67 93
11 65 58 38 28 38 21 16 22 41 76 41 30 25 68 58 55 62 69 20 13 72 34 94 28 31 46 100 14 33 45 99 23 63 77 84 54 84 35 39 89 98 66 63 83 31 11 12 61 78 84 44 40 57 86 94 67 17 55 32 42 90 67 57 37 48 59 87 67 86 57 88 46 13 72 76
51 39 88 14 28 73 20 49 71 32 62 67 36 37 74 10 47 12 46 84 85 97 66 93 46 65 71 62 18 89 48 73 32 26 95 69 76 73 99 26 45 68 74 18 68 86 35 92 30 96 16 67 69 96 77 25 20 19 99 30 13 20 41 85 66 96 96 32 16 85 35 44 29 72 50 64
67 81 46 24 71 19 11 82 85 39 76 23 51 98 37 40 70 70 42 74 63 12 24 42 15 32 41 45 37 46 59 31 35 61 11 52 74 44 37 64 70 17 70 50 29 94 45 94 54 45 65 58 46 82 73 40 73 61 70 23 28 71 72 75 76 49 75 87 80 20 88 88 36 44 47 67
94 88 48 32 30 64 57 95 71 74 41 15 49 30 10 42 61 99 82 55 15 47 87 76 70 69 42 24 25 72 37 30 71 89 58 82 87 21 80 39 92 51 40 34 31 37 45 71 73 78 66 53 86 66 60 51 93 73 46 46 15 43 46 43 87 52 52 67 42 31 85 52 39 20 11 21
71 49 94 78 56 57 100 21 92 10 98 96 51 42 13 36 64 72 72 96 74 39 73 51 62 78 17 65 39 98 76 54 38 47 38 81 81 91 83 81 97 10 38 58 67 98 50 68 93 52 91 35 82 69 22 83 79 10 96 21 89 59 84 44 24 30 46 92 17 91 21 47 97 83 96 55
13 56 31 63 14 99 17 92 16 39 74 46 58 86 19 25 21 41 20 82 74 38 67 96 80 75 95 31 48 17 71 77 16 68 73 21 78 77 83 61 69 79 94 27 41 65 83 88 94 35 78 59 46 61 91 99 68 98 71 11 61 79 99 13 35 47 82 99 43 66 54 37 28 75 51 50
24 71 74 32 66 57 65 33 81 26 27 54 20 17 74 70 23 49 47 60 23 41 68 38 50 43 98 14 26 92 81 14 95 27 58 91 69 29 88 31 36 78 37 28 73 97 38 98 63 89 59 85 64 82 75 82 19 93 12 43 35 11 24 36 28 60 12 65 87 87 73 70 84 27 15 76
68 24 12 82 37 97 59 99 81 54 39 81 70 93 63 95 15 80 99 58 12 38 31 75 29 35 89 20 32 18 16 23 44 90 33 82 10 93 90 56 60 48 42 79 18 78 45 82 82 62 55 31 54 93 100 85 36 44 37 81 50 44 73 44 94 35 81 46 97 65 50 100 89 96 63 95
92 64 67 14 67 21 33 87 20 48 31 53 91 34 21 68 62 92 42 58 79 87 67 56 39 83 94 64 15 18 73 89 18 34 41 81 53 27 26 96 58 74 86 20 90 71 58 85 67 57 44 51 41 73 33 34 56 57 67 66 44 88 29 78 96 49 95 48 74 25 55 99 82 36 95 83
44 98 77 87 63 64 16 53 70 100 50 94 24 25 44 54 33 98 17 21 78 77 32 32 47 22 40 49 69 82 98 30 29 21 76 31 72 76 95 98 65 62 76 18 70 15 15 30 55 70 81 65 22 92 88 81 95 25 75 83 15 52 72 59 81 54 11 32 47 99 37 78 77 96 18 57
70 11 38 68 67 29 20 49 67 63 69 41 60 22 68 40 51 99 89 12 53 79 57 96 62 26 91 81 52 15 100 31 35 16 66 46 36 26 38 27 16 90 90 41 86 72 14 24 74 79 74 18 56 88 91 75 80 35 21 31 81 74 83 64 17 90 25 42 52 58 87 16 53 63 69 50
15 44 44 92 94 35 63 41 69 44 81 46 76 39 55 14 82 93 31 39 30 99 24 59 15 74 80 21 63 71 91 79 82 56 94 86 49 76 99 22 50 27 21 26 73 32 96 46 57 96 62 81 72 15 63 57 82 42 37 33 55 71 30 79 23 72 12 61 74 96 59 74 71 65 68 17
82 21 31 70 59 67 67 45 53 56 18 10 43 37 70 86 90 37 43 33 94 93 59 57 71 10 86 17 18 12 53 23 66 66 69 42 16 60 85 44 19 20 24 95 33 28 62 49 77 10 49 70 31 78 28 14 87 31 89 69 28 93 46 61 34 84 80 36 89 97 57 80 100 95 52 76
39 61 73 69 73 11 72 21 83 83 92 85 53 21 93 63 24 98 25 96 40 51 52 58 35 99 68 30 73 29 54 86 51 38 92 38 47 60 29 23 44 23 83 58 49 70 98 88 73 67 73 88 26 30 50 73 35 69 48 36 51 76 62 62 36 50 85 49 73 21 100 20 54 72 35 91
81 81 11 71 100 20 36 38 44 68 99 57 87 76 90 63 56 44 23 33 29 24 51 33 12 65 42 48 75 33 82 69 19 30 77 12 22 27 88 13 92 96 48 23 61 65 29 45 94 51 12 19 96 60 91 69 80 40 42 57 18 10 27 80 11 60 49 27 14 65 11 99 73 10 54 41
39 68 100 32 42 53 83 14 75 22 76 100 76 76 58 44 73 87 55 66 54 16 61 27 90 51 32 32 28 67 30 42 62 71 40 64 52 98 90 84 97 74 70 64 53 55 57 23 24 66 92 48 62 44 98 75 65 41 79 53 97 66 13 52 29 69 98 80 27 41 58 22 20 74 40 92
68 66 82 77 50 80 83 65 55 39 33 79 77 86 29 83 13 11 76 24 79 58 40 48 90 60 43 82 65 10 25 45 13 69 71 24 91 30 71 86 69 60 59 94 98 60 72 60 24 54 74 69 84 94 11 21 28 60 53 69 18 24 32 93 47 26 35 57 80 76 91 58 68 45 18 78
38 75 91 12 15 21 29 27 15 73 20 35 30 81 60 79 23 46 82 38 42 63 16 59 93 86 30 92 14 55 52 100 20 20 14 52 89 32 18 55 14 97 72 99 10 44 14 23 99 25 91 42 36 61 14 11 57 91 40 10 70 33 64 10 89 59 41 85 32 74 51 60 94 18 72 39
96 81 68 17 97 63 99 46 80 24 82 54 63 58 61 68 93 30 77 88 99 63 60 100 11 48 61 95 14 60 56 53 21 81 98 65 67 18 83 62 68 13 21 99 99 51 22 67 56 45 99 72 90 82 73 79 65 20 87 12 48 61 74 87 88 46 70 80 19 64 25 48 30 81 31 35
