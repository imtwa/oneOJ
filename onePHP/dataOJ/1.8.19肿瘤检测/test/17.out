2551 2466
